app/views/school/welcome_wizard/what_happens_next.html.erb

```diff
@@ -20,7 +20,7 @@
         <li>change your Chromebook settings</li>
       </ul>
 
-      <%= f.govuk_submit (@current_user.has_multiple_schools? ? 'Finish and go to school page' : 'Finish and go to homepage' ) %>
+      <%= f.govuk_submit (@current_user.has_multiple_schools? ? "Finish and go to #{@school.name} overview" : "Finish and go to homepage" ) %>
```

Comment:
Schools names can be really long, this would be a big button

Reply:
Good point. I'll switch it to institution type, thanks